code/mobile/mobile-common/Utils/RenderUtils.cpp

```diff
@@ -114,7 +114,7 @@ Renderer::Renderer(vector<pair<GLuint, bool>> texs, string frag, float xStart, f
 
 Renderer::Renderer(int width, int height, vector<pair<GLuint, bool>> texs, string frag, GLenum fmt,
                    GLint mag, GLuint rdrBuf, int max) : width(width), height(height),
-                                                        size(width * height * 4), rtt(true), maxLod(max) {
```

Comment:
Why this change ?

Reply:
Change was done In order to fix compiler warnings but committed unknowingly.